config/city-lists.json

```
@@ -2,72 +2,122 @@
 	{
 		"name": "Online",
 		"country": " Edition",
-		"date": "October 30, 2024",
+		"date": "October 29, 2025",
 		"cfpdate":"Not announced yet",
 		"description": "Join us for the AsyncAPI Online Conference Edition as the community unites across the globe to share experiences, collaborate, and foster meaningful connections.",
 		"img": "/img/testMic.webp",
 		"address": "AsyncAPI YouTube Channel",
-		"map": "https://www.youtube.com/live/F9wHxd-v2f0?si=PT8BuAUKNmoLHRiM",
+		"map": "",
```

The conference will be hosted on the YouTube channel 
```suggestion
		"map": "https://www.youtube.com/@AsyncAPI",
```